task-buffer/task-buffer.test.ts

```diff
@@ -1,4 +1,4 @@
-import { run, sleep, spawn, type Task } from "effection";
+import { run, sleep, spawn, type Task } from "npm:effection@3.0.3";
```

Comment:
If we hard code to effection v3.0.3 in the import map, shouldn't this just be `"effection"`?